src/main/java/bio/terra/cli/businessobject/Server.java

```diff
@@ -57,6 +58,7 @@ public Server(PDServer configFromDisk) {
     this.wsmDefaultSpendProfile = configFromDisk.wsmDefaultSpendProfile;
     this.dataRepoUri = configFromDisk.dataRepoUri;
     this.externalCredsUri = configFromDisk.externalCredsUri;
+    this.idTokenAuthentication = configFromDisk.idTokenAuthentication;
```

Comment:
nit: to sound like a boolean, maybe name it `useIdTokenAuthentication`.

Reply:
Per some other comments, given this is a property on the server, perhaps it makes sense for it to be `acceptsIdToken` or `supportsIdToken` (dropping the `...Authentication` for brevity)?